python/lsst/daf/butler/registry/managers.py

```diff
@@ -204,6 +249,16 @@ def loadRepo(self, database: Database) -> RegistryManagerInstances:
         else:
             raise LookupError(f"Registry attribute {_DIMENSIONS_ATTR} is missing from database")
         universe = DimensionUniverse(dimensionConfig)
+        if self.obscore is not None:
+            # Get ObsCore configuration from attributes table, this silently
+            # overrides whatever may come from config file. Idea is that we do
+            # not want to carry around the whole thing, and butler config will
+            # have empty obscore configuration after initialization.
```

Comment:
We don't have that logic yet, I'll try to find a place where to add it.

Reply:
I implemented stripping of obscore config before writing `butler.yaml` on [8305317](https://github.com/lsst/daf_butler/pull/732/commits/8305317fbc57c5fb4d15c975f087fcd92fa7ebc1). I'm not super happy about it because it adds some knowledge of obscore to Butler class, but I could not figure out a better place for that logic.